portal/api-tests/utilisation-report-routes.api-test.js

```diff
@@ -128,4 +128,47 @@ describe('utilisation-report routes', () => {
       });
     });
   });
+
+  describe('POST /utilisation-reports/cancel-correction/:correctionId', () => {
+    const originalProcessEnv = { ...process.env };
+    const getUrl = ({ correctionId }) => `/utilisation-reports/cancel-correction/${correctionId}`;
+
+    describe('when FF_FEE_RECORD_CORRECTION_ENABLED is set to `true`', () => {
+      beforeAll(() => {
+        process.env.FF_FEE_RECORD_CORRECTION_ENABLED = 'true';
+      });
+
+      afterAll(() => {
+        process.env = { ...originalProcessEnv };
+      });
+
+      withRoleValidationApiTests({
+        makeRequestWithHeaders: (headers) => post().to(getUrl({ correctionId: 1 }), {}, headers),
```

Comment:
Is this correct syntax? Does `to` not just accept the url and we provide the body and headers to `post` ?

As in (untested):
```suggestion
        makeRequestWithHeaders: (headers) => post({}, headers).to(getUrl({ correctionId: 1 })),
```

Reply:
`portal/api-tests/create-api.js:10` looks like this isn't supported; `to` is defined as: `to: async (url) =>` ...

Not sure why this is passing given `withRoleValidationApiTests` passes headers in?? - not investigated

Reply:
Hmm that is odd, this does look wrong, I will investigate why it even passes!